language/de/acp/board.php

```diff
@@ -353,6 +353,8 @@
 
 	'COOKIE_DOMAIN'				=> 'Cookie-Domain',
 	'COOKIE_NAME'				=> 'Cookie-Name',
+	'COOKIE_NOTICE'				=> 'Cookie-Hinweis',
+	'COOKIE_NOTICE_EXPLAIN'		=> 'Wenn aktiviert, wird den Besuchern deines Boards ein Cookie-Hinweis angezeigt. Dies kann abhängig der in deinem Land geltenden rechtlichen Regelungen, dem Inhalt deines Boards und den aktivierten Erweiterungen notwendig sein.',
```

Comment:
Wenn dann "von den in deinem Land geltenden" - meiner Ansicht nach passt die Variante mit der aber